src/app/prt-staking/components/prt-section/widget-header.tsx

```diff
@@ -1,9 +1,8 @@
 import { XMarkIcon } from '@heroicons/react/16/solid'
-import { TokenData } from '@indexcoop/tokenlists'
 import Image from 'next/image'
 
 type Props = {
-  tokenData: TokenData
+  tokenData: { logoURI: string; symbol: string }
```

Comment:
you could actually stuff like this here:
<img width="343" alt="Screenshot 2024-10-11 at 11 38 11" src="https://github.com/user-attachments/assets/df4e8c4f-5150-4fa3-85df-adc41dd58854">
<img width="872" alt="Screenshot 2024-10-11 at 11 38 20" src="https://github.com/user-attachments/assets/d81c1589-c02b-41bd-a241-8fa74dd7356e">